crates/dojo-core/src/config/component.cairo

```diff
@@ -0,0 +1,59 @@
+mod errors {
+    const INVALID_CALLER: felt252 = 'Config: not owner or operator';
+    const ALREADY_REGISTERED: felt252 = 'Config: already operator';
+    const NOT_OPERATOR: felt252 = 'Config: not operator';
+}
+
+#[starknet::component]
+mod Config {
+    use dojo::config::interface::IConfig;
+    use starknet::ContractAddress;
+    use super::errors;
+    use starknet::get_caller_address;
+
+    #[storage]
+    struct Storage {
```

Comment:
What do you think about using dojo storage model here? Then we will get acl and indexing for free

Reply:
@neotheprogramist we should also emit an event to once the state has been updated. 👍